src/presence_analyzer/views.py

```diff
@@ -75,3 +78,30 @@ def presence_weekday_view(user_id):
 
     result.insert(0, ('Weekday', 'Presence (s)'))
     return result
+
+
+@app.route('/api/v1/presence_start_end/<int:user_id>', methods=['GET'])
+@jsonify
+def presence_start_end(user_id):
+    """
+    Calculate average time when user start the work and when user end the work.
+    """
+    data = get_data()
+    if user_id not in data:
+        log.debug('User %s not found!', user_id)
+        abort(404)
+
+    resul = {}
+    for item in data[user_id]:
```

Comment:
It wont return full days list of week if someone wasn't working full week. Display all days even if there wasnt any entries